cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -3577,6 +3577,22 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    ig.push(
+        Inst::new(
+            "fmin_pseudo",
+            r#"
+        Floating point pseudo-minimum, propagating NaNs.
+        See https://github.com/WebAssembly/simd/pull/122 for background.
+
+        If either operand is NaN, this returns a NaN.  If both operands are zeroes, regardless
+        of sign, the first operand (``x``) is returned.
```

Comment:
Based on the referred PR and tests cases at simd_XX_pmin_pmax.wast's , the NaN behavior is different. I recommend to copy `pmin(a, b) := b < a ? b : a` definition language here is extract NaN propagation from it.